packages/venia-concept/src/components/MiniCart/kebab.css

```diff
@@ -0,0 +1,55 @@
+.root {
+    position: relative;
+    display: inline-block;
+
+    grid-row-start: 1;
+    grid-row-end: 4;
+    grid-column-start: 3;
+    grid-column-end: 4;
+    align-self: center;
+}
+
+.root ul {
+    z-index: 3;
+    position: absolute;
+    right: 0.5em;
+    box-shadow: 0 0 1px #999;
+}
+
+.kebab {
+    outline: 0;
+    border: none;
+    background-color: #fff;
+}
+
+.dropdown {
+    display: grid;
+    position: absolute;
+    right: 0;
+    top: -2.5em;
+    transition: all 0.25s ease-out;
+    transform: scale(0);
+    transform-origin: 100% 50%;
+    align-items: center;
+}
+
+.dropdown.ul:hover {
+    transform: scale(1);
+}
+
+.dropdown li {
+    display: block;
+    width: 100%;
+    background-color: #fff;
+    border-bottom: 1px solid rgb(var(--venia-border));
+}
+
+.dropdown li:hover {
+    background-color: #eee;
+}
+
+.active {
+    position: absolute;
+    transform: scale(1);
+    transition: all 0.25s cubic-bezier(0.5, 1.8, 0.9, 0.8);
```

Comment:
This is a really nice feeling transition. Maybe we should put it in a central place, so that other appear/disappear actions can have a similar feel.

Have you tested it on a low-poered device to make sure its framerate is ok?

Reply:
As far as devices go, it's only been tested using the Chrome performance tab. With 6x CPU slowdown and some artificial performance overhead, I didn't notice any significant difference. If you have a target device in mind and want to take a look, the checkout options branch is live at https://upward.bargreen.io

Reply:
@zetlen Right now we're only using CSS custom properties when we want to store reusable values. Timing functions could certainly be stored as such, though.

@Serunde Consider limiting `transition-property` to the properties being transitioned, rather than using `all`. 😅 

```css
/* index.css */
html {
  --venia-anim-bounce: cubic-bezier(0.5, 1.8, 0.9, 0.8);
}

/* kebab.css */
.myElement {
  transition: transform 256ms var(--venia-anim-bounce);
}
```